src/Planners/include/BipedalLocomotion/Planners/SwingFootPlanner.h

```diff
@@ -75,10 +75,10 @@ class SwingFootPlanner : public System::Source<SwingFootPlannerState>
      * |        `sampling_time`       | `double` |                                        Sampling time of the planner in seconds                                       |    Yes    |
      * |         `step_height`        | `double` | Height of the swing foot. It is not the maximum height of the foot. If apex time is 0.5 `step_height` is the maximum |    Yes    |
      * |        `foot_apex_time`      | `double` |   Number between 0 and 1 representing the foot apex instant. If 0 the apex happens at take off if 1 at touch down    |    Yes    |
-     * |    `foot_landing_velocity`   | `double` |                                               Landing vertical velocity                                              |    Yes    |
-     * |  `foot_landing_acceleration` | `double` |                                             Landing vertical acceleration                                            |    Yes    |
-     * |   `foot_take_off_velocity`   | `double` |                                              Take-off vertical velocity                                              |    Yes    |
-     * | `foot_take_off_acceleration` | `double` |                                            Take-off vertical acceleration                                            |    Yes    |
+     * |    `foot_landing_velocity`   | `double` |                                               Landing vertical velocity                                              |    No    |
+     * |  `foot_landing_acceleration` | `double` |                                             Landing vertical acceleration                                            |    No    |
+     * |   `foot_take_off_velocity`   | `double` |                                              Take-off vertical velocity                                              |    No    |
+     * | `foot_take_off_acceleration` | `double` |                                            Take-off vertical acceleration                                            |    No    |
```

Comment:
Could you add the default value in the documentation?

Reply:
Done in b3ccb3e